scripts/npm/check-owners.js

```diff
@@ -119,7 +119,7 @@ const verifyMaintainers = data => {
 console.log(`Maintainers are:\n  ${owners.join('\n  ')}\n`);
 
 readdir('packages/')
-  .then(packages => packages.filter(i => !i.startsWith('.')))
+  .then(packages => packages.filter(i => i !== 'bpk-component-boilerplate'))
```

Comment:
Shouldn't we read whether the package is private from package.json?

Maybe that's over-engineering. To make this more readable we could use 
```
const privatePackages = ['bpk-component-boilerplate'];
...
.then(packages => packages.filter(i => !privatePackages.contains(i)))
```